docs/02-admin/03-optional-features/07-anubis.md

```diff
@@ -0,0 +1,408 @@
+# Anubis setup for mbin
+
+### Why?
+
+Anubis is a bot programm trying to block bots by presenting proof-of-work challengers to the user. A normal browser will just solve them (provided JavaScript is enabled on it) whilst AI scrapers will most likely just accept the challenge as the response.
+
+Ths simple answer is: because it is better than just blocking anonymous access to Mbin.
+
+### How does it work?
+
+https://anubis.techaro.lol/docs/design/how-anubis-works
```

Comment:
Should we make an actual markdown link for this? So we are sure it becomes a proper html link?